book-hu/chapter10.md

```
@@ -0,0 +1,738 @@
+> - Két Fordító párhuzamosan lefordítja a szójegyzéket németre és japánra.
+> - A Műszaki Felülvizsgáló ellenőrzi a lefordított dokumentáció műszaki pontosságát.
+> - A Formázó egységesíti a formázást.
+> - Végül a tesztelő integrációs teszteket futtat.>
+
+>> **2. lépés: Al-ügynök Csoport Létrehozása**. A `triage` átadja a kontextust a Menedzser Ügynöknek, amely létrehozza és ütemezi a feladatokat. A kód stílusának szemléltetésére:>
+
+>> ```python
+> # Szójegyzék feladat: indíts egy al-ügynököt a szójegyzék létrehozásához
+> task_glossary = spawn_subagent(
+>     agent_id="glossary_writer",
+>     system_prompt="Angol műszaki szójegyzék írója. {glossary_rules}",
```

markdownlint-cli2 (0.23.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)